services/QuillLMS/app/services/vitally_integration/serialize_vitally_sales_user.rb

```diff
@@ -100,7 +105,14 @@ def data
         email_verification_status: @user.email_verification_status,
         admin_approval_status: @user.admin_approval_status,
         number_of_schools_administered: number_of_schools_administered,
-        number_of_districts_administered: number_of_districts_administered
+        number_of_districts_administered: number_of_districts_administered,
+        learn_worlds_num_enrolled_courses: learn_worlds_enrolled_courses&.count,
+        learn_worlds_enrolled_courses: learn_worlds_enrolled_courses&.titles_string,
```

Comment:
Do we have a sense of how big the list of enrolled courses might eventually get?  I imagine that there's some size at which passing a giant list of strings to Vitally is a bad idea.  Less than 10 should be fine, but how close to 100 do we want to allow this to comfortably get?

Reply:
That's a good question.  I was under the impression this would be under 10.

Reply:
I'll set a reminder to check every few months how many courses are in Quill Academy.